css/styles.css

```diff
@@ -0,0 +1,17 @@
+@import url("_header.css");
+@import url("_fonts.css");
+
+body{ font-family: "Encode Sans", sans-serif; background: #ffffff; margin: 0px; font-size: 14px;}
```

Comment:
Because I see you copy-pasted this CSS code from the index.html file, I guess you did it some time ago.
There are a bunch of things you should refactor here:
1. CSS blocks should be spread on multiple lines, and not everything in one line. It's really hard to read like this...
It should be formatted like so (notice my new-lines, white-spaces, colons and semi-colons).
```
a:link {
    color: #41b7d8;
}
```

2. You are styling tags. Now that you know CSS classes, you should use them to style elements.